package update, regex update and using in devcontainer

dbrandes-welfenlab opened the pull request:
Hi,

thanks for this great extension for code. It makes documentation of code much better. I need it in devcontainers and want to use it together with the drawio-extension (use .drawio.svg and .drawio.png as extensions), so I need the fixes from this request. It would be cool if you would integrate this into your code and push a new release to the vscode extension repo.

best regards,

Daniel

dbrandes-welfenlab committed:
package-lock.json,
package.json:

 - updated packages and removed not needed engines.

dbrandes-welfenlab committed:
package.json,
src/extension.ts:

 - updated to 0.0.2
 - fixed publisher field to what vs code wants there.
 - added two parameters for using with dev containers: commentimg.workspace_path_override and commentimg.home_dir_override
 - modified regex to allow svg files, dots and minus in filenames.

dbrandes-welfenlab committed:
src/extension.ts:

 - bugfix to use variables from settings right.

dbrandes-welfenlab committed:
README.md:

 - updated documentation.

dbrandes-welfenlab committed:
README.md:

 - spelling.

vnermolaev commented on README.md:
```diff
@@ -11,8 +11,29 @@ Inside a comment write
+
+### devcontainer
+
+to use in devcontainer, two options are there:
```
Would you start sentences with a capital letter?

dbrandes-welfenlab replied:
Hi,

thanks for the fast reply and the comments. I tried to fix them.

I fixed this, sorry for the spelling issues. If you find more, please write again or fix it by yourself if you like.

vnermolaev commented on package.json:
```diff
@@ -2,8 +2,8 @@
-    "version": "0.0.1",
-    "publisher": "Victor Ermolaev",
+    "version": "0.0.2",
+    "publisher": "quicky84",
```
Please keep the original name

dbrandes-welfenlab replied:
I have changed it back, sorry for this. I locally set it to your vscode name (hope this was you), because I could not package it with the vs code tools to test it in devcontainer.

vnermolaev commented on package.json:
```diff
@@ -20,13 +20,35 @@
+						"string"
+					],
+					"default": "",
+					"description": "Override path to workspace to display images from. Leave Empty to use default one. Useful for devcontainers. this is relative to homedir if commentimg.home_dir_override is set.",
```
Would you align these descriptions with those in README?

dbrandes-welfenlab replied:
I have joined these and now these are the same (hope you mean this.).

Additionally I have another small request to you:

Can you add a licence file to your extension?

best regards,

Daniel

vnermolaev reviewed (changes requested):
Thanks for your effort! I have requested only a few small changes.

dbrandes-welfenlab committed:
CHANGELOG.md,
README.md,
package.json:

 - Updated changelog,
 - reverted publisher changes and
 - updated and synchronized documentation.
